CONTRIBUTING.md

```diff
@@ -1,4 +1,73 @@
-Contributing
-======
+# How to contribute
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md) in the Home repo.
+One of the easiest ways to contribute is to participate in discussions and discuss issues. You can also contribute by submitting pull requests with code changes.
+
+
+## General feedback and discussions?
+Please start a discussion on the [repo issue tracker](https://github.com/aspnet/AspNetCore/issues).
+
+
+## Bugs and feature requests?
+For non-security related bugs please log a new issue in the appropriate GitHub repo. Here are some of the most common repos:
+
+* [Docs](https://github.com/aspnet/Docs)
+* [EntityFrameworkCore](https://github.com/aspnet/EntityFrameworkCore)
+* [Tooling](https://github.com/aspnet/Tooling)
+* [Extensions](https://github.com/aspnet/Extensions)
+
+Or browse the full list of repos in the [aspnet](https://github.com/aspnet/) organization.
+
+## Reporting security issues and bugs
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC)  secure@microsoft.com. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://technet.microsoft.com/en-us/security/ff852094.aspx).
+
+
+## Other discussions
+Our team members also monitor several other discussion forums:
+
+* [ASP.NET Core forum](https://forums.asp.net/1255.aspx/1?ASP+NET+5)
+* [Stack Overflow](https://stackoverflow.com/) with the [`asp.net-core`](https://stackoverflow.com/questions/tagged/asp.net-core), [`asp.net-core-mvc`](https://stackoverflow.com/questions/tagged/asp.net-core-mvc), or [`entity-framework-core`](https://stackoverflow.com/questions/tagged/entity-framework-core) tags.
+
+
+## Filing issues
+When filing issues, please use our [bug filing templates](https://github.com/aspnet/AspNetCore/wiki/Functional-bug-template).
```

Comment:
Do you want to remove (or heavily edit) this section, since you're adding proper issue templates?

Reply:
Sounds good to me. Removed this section in favor of issue templates.